.github/workflows/storage-node.yml

```diff
@@ -0,0 +1,38 @@
+name: storage-node
+on: [pull_request, push]
+
+jobs:
+  storage_node_build_ubuntu:
+    name: Ubuntu Checks
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: checks
+      run: |
+        yarn install --frozen-lockfile
+        yarn workspace storage-node checks
+
+  storage_node_build_osx:
+    name: MacOS Checks
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: checks
+      run: |
+        yarn install --frozen-lockfile --network-timeout 120000
+        yarn workspace storage-node checks
+        yarn workspace @joystream/storage-cli build
```

Comment:
This one isn't part of the Ubuntu job, it also doesn't seem necessary, because `yarn workspace storage-node run build` (which does the same thing) is part of root `package.json` `postinstall` script already (so it's beeing run in each of the jobs after `yarn install`).

Reply:
good point, will remove

Reply:
also I thought I'd already pushed a second commit to add it to ubuntu job :)

Reply:
fixed in 000e2db0157d1bb6e876c37a1c256d81d2cd6305